### PR TITLE
Update the product of reserve balances at the end of converter upgrade

### PR DESCRIPTION
**Pull request by barakman**

### CODE REVIEW

File: solidity/contracts/converter/ConverterBase.sol [hunk 1]
```diff
@@ -258,6 +258,18 @@ abstract contract ConverterBase is ConverterVersion, IConverter, ContractRegistr
         acceptOwnership();
     }
 
+    /**
+     * @dev does nothing
+     */
+    function postUpgrade()
```
Comment (yudilevi) [inline solidity/contracts/converter/ConverterBase.sol:264]: -> `onUpgradeComplete`

File: solidity/contracts/converter/ConverterBase.sol [hunk 2]
```diff
@@ -258,6 +258,18 @@ abstract contract ConverterBase is ConverterVersion, IConverter, ContractRegistr
         acceptOwnership();
     }
 
+    /**
+     * @dev does nothing
```
Comment (yudilevi) [inline solidity/contracts/converter/ConverterBase.sol:262]: "executed by the upgrader at the end of the upgrade process to handle and custom pool logic"

**Review by yudilevi — CHANGES_REQUESTED** [review]

File: solidity/contracts/converter/ConverterBase.sol (hunk 2 above)
Comment (barakman) [reply solidity/contracts/converter/ConverterBase.sol]: And in the StandardPoolConverter?

File: solidity/contracts/converter/ConverterBase.sol (hunk 1 above)
Comment (barakman) [reply solidity/contracts/converter/ConverterBase.sol]: Done

File: solidity/contracts/converter/ConverterBase.sol (hunk 2 above)
Comment (yudilevi) [reply solidity/contracts/converter/ConverterBase.sol]: oops, had a typo there ("and")

Comment (yudilevi) [reply solidity/contracts/converter/ConverterBase.sol]: and can use it anywhere

Comment (barakman) [reply solidity/contracts/converter/ConverterBase.sol]: Done

**Review by yudilevi — APPROVED** [review]

**Pull request merged**